CHANGELOG.md

```diff
@@ -36,6 +36,7 @@ Jsonnet users will now need to specify a storage request and limit for the gener
         ephemeral_storage_limit_size: '11Gi',
       },
     }
+* [CHANGE] Wider default latency buckets for metrics-generator spanmetrics. [#1593](https://github.com/grafana/tempo/pull/1593) (@fredr)
```

Comment:
```suggestion
* [CHANGE] Two additional default buckets added to the generated span metrics. This will add cardinality when using the default setting. [#1593](https://github.com/grafana/tempo/pull/1593) (@fredr)
```
Good point @kvrhdn, something like this maybe?

Reply:
I'll commit this change, let me know if you think I should phrase it differently.